geomstats/geometry/special_euclidean.py

```diff
@@ -88,11 +91,15 @@ def get_identity(self, point_type=None):
             point_type = self.default_point_type
 
         identity = gs.zeros(self.dimension)
-        if self.default_point_type == 'matrix':
-            identity = gs.eye(self.n)
+        if point_type == 'matrix':
+            identity = gs.eye(self.n + 1)
         return identity
     identity = property(get_identity)
 
+    def get_dimension(self):
+        """Get the dimension of each object given the default_point_style."""
```

Comment:
default_point_style -> default_point_type, or point_type?

Reply:
Also, this should be renamed, as it does not really return the `dimension` of the Lie group: we reserve this name for the actual dimension of SE(n), which is n(n + 1)/2.

Maybe `get_point_shape`? (It's not great, happy to choose something else if you have a better idea!)

Reply:
- That's a good question. With the way things are coded now, most functions do not specify point_type, therefore relying on the default_point_type.
- Peut-être `get_representation_size`, pour bien appuyer sur le fait que c'est lié à la représentation ?

Reply:
Seems like Skyping in French and writing in English do not go well along. I had thought of `get_representation_size`to emphasize that it was linked to the chosen representation. Why not be even more direct and say `get_point_type_shape` with `point_type` as an optional argument ?

Reply:
- I would suggest self.point_type (no need for the default_*), since - as you said - point_type is more often used as the object's attributes than as a method's parameter. Then, methods should have `point_type=None` as an optional parameter, and within the method's body: `if point_type is None: point_type = self.point_type`.
- `size` is tricky, because in numpy, `np.size(my_array)` returns the total number of elements in the array, so we should prefer `shape` there. I like `get_point_type_shape`! I think it's better to be over-explicit for now indeed.

Reply:
In the way the things are right now, there is no `point_type` attribute, only the default one. So I'll stick to this one, with `point_type` as optional argument, with None as base value.